style/train/classifier_trainer.py

```diff
@@ -1,28 +1,135 @@
-from typing import List
+from pathlib import PosixPath
```

Comment:
is there more general `Path`? Why do we use `PosixPath`?

Reply:
After I added PosixPath as typing, the GUI added this automatically.